.github/workflows/upstream-dev-ci.yml

```diff
@@ -2,10 +2,10 @@ name: CI Upstream
 on:
   push:
     branches:
-      - main
+      - master
```

Comment:
could be separate PR to rename master to main
https://github.com/github/renaming#renaming-existing-branches
Looks simply and hopefully we'll remember to change this as well as other mentions of master in the docs